### DISCUSSION TITLE
seeker issues

### DISCUSSION THREAD

**Discussion by ikaggii** [community/community#37706] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/101791936/198826794-f8ad2612-9fcc-4f4e-bdac-fae02627afb3.png)
seeker shpowing error.. even on phone it taking me on ngork website.
![2022-10-29 16_09_01-](https://user-images.githubusercontent.com/101791936/198826860-ceb3a97d-2374-4556-ba27-71a43b46ce92.png)

**Comment by ossanna16** [comment]
Hi @ikaggii,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Seeker. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
